libs/community/tests/integration_tests/vectorstores/test_cratedb.py

```diff
@@ -495,9 +491,9 @@ def test_cratedb_retriever_search_threshold() -> None:
 
     retriever = docsearch.as_retriever(
         search_type="similarity_score_threshold",
-        search_kwargs={"k": 3, "score_threshold": 0.999},
+        search_kwargs={"k": 3, "score_threshold": 0.35},  # Original value: 0.999
```

Comment:
That's certainly an anomaly that had to be applied to an input value, in order to get the expected result here.